README.md

```diff
@@ -105,6 +105,7 @@ Which will produce:
 | **verbose**              | Pipe out more verbose output to the shell | CORDOVA_VERBOSE    | false |
 | **team_id**              | The development team (Team ID) to use for code signing  | CORDOVA_IOS_TEAM_ID               | *28323HT* |
 | **provisioning_profile** | GUID of the provisioning profile to be used for signing | CORDOVA_IOS_PROVISIONING_PROFILE  |           |
+| **package_type**         | This will determine what type of Android build is generated. <br>Valid options are apk and bundle | CORDOVA_ANDROID_PACKAGE_TYPE    | apk |
```

Comment:
```suggestion
| **package_type**         | This will determine what type of Android build is generated. Valid options are `apk` and `bundle` | CORDOVA_ANDROID_PACKAGE_TYPE    | apk |
```